spec/controllers/issues_controller_spec.rb

```diff
@@ -0,0 +1,92 @@
+require "rails_helper"
+RSpec.describe IssuesController, type: :controller do
+  describe "#index" do
```

Comment:
Tiny linting nitpick - we probably want a new line here before the it block

Reply:
Thanks for the suggestion! I tried adding a new line, but it triggered a RuboCop alert "Extra empty line detected at block body beginning." Should we adjust our RuboCop settings, or stick to its current rules for consistency?